app/user/auth-checker-user-only-filter.js

```diff
@@ -14,14 +14,14 @@ const authCheckerUserOnlyFilter = (req, res, next) => {
     .then(() => next())
     .catch(error => {
       if (error.name === 'FetchError') {
-        console.error(error);
+        // console.error(error);
```

Comment:
Similar to previous reviews above, please copy the approach to logging from https://github.com/hmcts/ccd-admin-web/blob/master/src/main/user/auth-checker-user-only-filter.ts.

Reply:
DONE